File: problems/0485-max-consecutive-ones/solution.c
int findMaxConsecutiveOnes(int* nums, int numsSize) {
    int c=0,max=0;

    for(int i=0;i<numsSize;i++){
        if(nums[i]==1){
            c++;
          if(c>max){
            max=c;
        }    
    }
        else {
            c=0;
        }
}
    return max;
}

File: problems/0485-max-consecutive-ones/solution.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxcount=0,c=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]==1){
                c++;
            }

            else{
                c=0;
            }
            maxcount=Math.max(maxcount,c);
        }

        return maxcount;
    }
}
